libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -178,16 +181,18 @@ class RSCGovFloatParam {
     static const struct AP_Param::GroupInfo var_info[];
 
     void set_gov_enable(int8_t setenable) {enable = setenable; }
-    int16_t get_setpoint() { return setpoint; }
+    int16_t get_reference() { return reference; }
+    float get_range() { return range; }
     float get_disengage() { return disengage; }
-    float get_droop_setting() { return droop_setting; }
+    float get_droop_response() { return droop_response; }
     float get_tc() { return tc; }
 
 private:
     AP_Int8   enable;
-    AP_Float  disengage;    // sets the throttle percent where the governor disengages for return to flight idle
-    AP_Int16  setpoint;     // sets rotor speed for governor
-    AP_Float  droop_setting;// governor response to droop under load
-    AP_Float  tc;           // governor throttle curve weighting, range 50-100%
+    AP_Int16  reference;      // sets rotor speed for governor
+    AP_Float  range;          // RPM range +/- governor rpm reference setting where governor is operational
+    AP_Float  disengage;      // sets the throttle percent where the governor disengages for return to flight idle
+    AP_Float  droop_response; // governor response to droop under load
+    AP_Float  tc;             // governor throttle curve weighting, range 50-100%
```

Comment:
perhaps "tc" could also be made more specific.  I would normally assume "TC" meant "time constant".  maybe "thr_curve_weighting".  Doesn't need to be really long but a bit longer to help the next developer guess what it's for without looking at the declaration comments would be good.

Reply:
This is now declared as _governor_thrcurve